src/main/docker/Dockerfile

```diff
@@ -32,3 +150,5 @@ LABEL Build.Number="$BUILD_NUMBER" \
       Git.Repo="$GIT_REPO" \
       Git.Branch="$GIT_BRANCH" \
       Git.Commit="$GIT_COMMIT"
+
+CMD ["python3"]
```

Comment:
Tagging the image should come last so move this above the `# Tag the image` comment.

Reply:
done